runtime/doc/news.txt

```diff
@@ -89,7 +89,13 @@ PLUGINS
 
 TREESITTER
 
-• TODO
+• |Query:iter_matches()| now defaults to returning all matching nodes in a
+  match, rather than simply the last node. In particular, the `match` table
+  returned by |Query:iter_matches()| now maps capture IDs to a list of nodes,
+  rather than a single node. The old behavior is incorrect, but can be opted
+  into by setting `all=false` in the `opts` argument to
+  |Query:iter_matches()|. This option will be removed completely in a future
+  release.
```

Comment:
```suggestion
• |Query:iter_matches()| correctly returns all matching nodes in a match
  instead of only the last node. This means that the returned table maps
  capture IDs to a list of nodes that need to be iterated over. For
  backwards compatibility, an option `all=false` (only return the last
  matching node) is provided that will be removed in a future release.
```

Reply:
Changed as suggested.